code/renderers/svelte/tsconfig.json

```diff
@@ -1,9 +1,11 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
+    "module": "ESNext",
+    "skipLibCheck": true,
     "strict": true,
     "resolveJsonModule": true
   },
   "include": ["src/**/*"],
-  "exclude": ["src/**/*.test.*"]
+  "exclude": []
```

Comment:
unless this has an effect, I'd remove this line.

Reply:
I think It's likely required, because of:
https://github.com/storybookjs/storybook/blob/962ed90ccc42509df081a6b969ca6965994b4143/code/tsconfig.json#L20-L29